uhyve-interface/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "uhyve-interface"
```

Comment:
I had this before, but changed it to `uhyve-interface` so that it can contain other interfaces as well, which are not explicitly hypercalls, like the UART (which is now a hypercall anyway :upside_down_face:  )

Reply:
How about `uhyve-guest` or `uhyve-guest-api`, then?

How is the UART not a hypercall? What kind of other mechanisms are in scope of this crate?